packages/evm/contracts/court/AragonCourt.sol

```diff
@@ -86,15 +83,26 @@ contract AragonCourt is Controller, IArbitrator {
     */
     function createDispute(uint256 _possibleRulings, bytes calldata _metadata) external returns (uint256) {
```

Comment:
⚠️ Note that at the moment, an arbitrable would not be able to call `rule()` if the DisputeManager module was changed

Reply:
Addressed in https://github.com/aragon/protocol/pull/19/commits/8f6c97ac5ee6931a685988520d07d6e393c6c0a3